README.md

```diff
@@ -54,6 +53,66 @@ VitePWA({
 })
 ```
 
+### Prompt for new content 
+
+![](https://user-images.githubusercontent.com/11247099/110332062-d726fa80-805a-11eb-92f4-771499241350.png)
```

Comment:
@antfu we need to capture the image again, I have changed its text.

And F5 will not activate the service worker, it requires a hard refresh, so I just configure this text:
`New content available, click on reload button to update.`